multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -26,6 +26,24 @@ def medium_quality_faster_train():
     }
 
 
+@automm_presets.register()
+def medium_quality_faster_inference_image_classification():
+    return {
+        "model.names": ["timm_image"],
+        "model.timm_image.checkpoint_name": "mobilenetv3_large_100",
```

Comment:
Just curious, is `model.names` required when `timm_image.checkpoint_name` is specified?

Reply:
I don't think it is required, correct me if I'm wrong @sxjscience @zhiqiangdon 

I see all other presets have `model.names` written out even though it is clear from the context, so I just follow.

Reply:
Better to keep the `model.names=["timm_image"]` to make the context more clear.

Reply:
+1, so let's keep the `model.names`.

Reply:
Thanks for the insights!